README.adoc

```diff
@@ -7,34 +7,52 @@ image::https://travis-ci.org/hawkular/hawkular-apm.svg?branch=master[Build Statu
 endif::[]
 
 [.lead]
-*Hawkular APM* is the Application Performance Management (APM) module for Hawkular.
+is the *Application Performance Management* solution offering:
 
-The project will provide the capabilities to monitor the flow of invocations
-across servers, tiers, on-premises and in the cloud. It will also enable detailed
+* Distributed Tracing (e.g. for microservices)
+* Application Performance Management
+* Business Transaction Management (currently only using JVM javaagent based approach)
+
+Applications can be instrumented, using a non-intrusive Java agent approach for JVM based applications, or through
+http://zipkin.io/pages/existing_instrumentations.html[Zipkin client libraries] (for polyglot environment).
+
+In other words Hawkular-apm provides the capabilities to monitor the flow of invocations
+across servers, tiers, on-premises and in the cloud. It also enables detailed
 performance analysis to be performed of the individual components that make up an
 application.
 
-== Build
+== Build & Run
 
-To build the project, run:
+To build and run execute:
 
 ```shell
-mvn clean install [ -Ptests ]
+$ mvn clean install -Pdev -DskipTests [-Ptests]
+$ dist/target/hawkular-apm-${version}/bin/standalone.sh [-Djboss.socket.binding.port-offset=1331]
+-> go to http://localhost:[8180|9411]/ and log in with jdoe:password
```

Comment:
Issue is the -Doffset is shown as optional, with the value for the zipkin port - so if not specified, the server default port is 8080 not 8180.
So either we need to make the -Doffset not optional, but instead have values 100 or 1331, or we need to change the project to use 8080 as its default port, and then users have to worry about port conflicts?